a10_neutron_lbaas/db/models.py

```diff
@@ -53,24 +59,47 @@ def find_all(cls, db_session=None):
         return cls.query(db_session).all()
 
     @classmethod
+    def create(cls, **kwargs):
```

Comment:
Are you certain that these generic method names don't already exist, which would lead to later weirdness?